modules/core/src/main/java/org/apache/synapse/config/xml/endpoints/AddressEndpointFactory.java

```diff
@@ -59,6 +62,8 @@
  */
 public class AddressEndpointFactory extends DefaultEndpointFactory {
 
+    private static final String SYSTEM_VARIABLE_PREFIX = "$SYSTEM";
```

Comment:
have we used this constant anywhere in the code?

Reply:
Used in SynapseConstants class. Will remove unused reference